docs/general/administration/hardware-acceleration/apple.md

```diff
@@ -0,0 +1,102 @@
+
+## Acceleration Methods
+
+[VideoToolbox](https://developer.apple.com/documentation/videotoolbox) is the only available method on macOS.
+
+To achieve full acceleration, [Metal](https://developer.apple.com/metal/) is required.
+
+## Tone-mapping Methods
+
+Hardware accelerated HDR to SDR tone-mapping is supported on all Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+There are two different methods that can be used. Pros and cons are listed below:
+
+1. **Metal**
+
+   - Pros: Supports Dolby Vision P5, detailed fine-tuning options.
+
+   - Cons: Slower on low end GPU, especially on Intel iGPUs.
+
+2. **VideoToolbox Native**
+
+   - Pros: Lower power consumption, less dependency on GPU performance, good visual quality without manual fine-tuning.
+
+   - Cons: Lack of tuning options, does not support Dolby Vision P5.
+
+When both methods are enabled, VideoToolbox Native will be used for most videos, and Metal will only be used as a fallback for Dolby Vision Profile 5 videos.
+
+## Select System Hardware
+
+Hardware accelerated transcoding is supported on all Macs that support [VideoToolbox](https://developer.apple.com/documentation/videotoolbox). This includes most 2011 and later Macs.<sup>*</sup>
+
+Full acceleration is available on most Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+To have native Apple Silicon support, Jellyfin server 10.9.0+ and `jellyfin-ffmpeg6` 6.0.1-5 or higher is required.
+
+While hardware acceleration via VideoToolbox might work on older series Macs, it is not officially supported.
+
+<sup>*</sup> VideoToolbox may fallback to software decoding/encoding or stop working when the task exceeds its hardware capability. The older your Mac, the more this will occur. Macs from 2017 and later are strongly recommended.
+
+### Transcode H.264
+
+AVC / H.264 8-bit is still widely used due to its excellent compatibility. Most GPUs after 2011 can decode and encode it.
+
+Any VideoToolbox-supported Mac supports decoding and encoding H.264 8-bit.
+
+### Transcode HEVC
+
+HEVC / H.265 remains the first choice for storing 4K 10-bit, HDR and Dolby Vision video. It has mature software encoding support thanks to [x265](https://x265.readthedocs.io/en/master/), as well as the widely implemented hardware encoding support in most GPUs released after 2016.
+
+Macs from 2017 and later, excluding the MacBook Air (13-inch, 2017), support decoding and encoding HEVC.
+
+### Transcode AV1
+
+There is no hardware accelerated path for AV1 on macOS at the moment. Although the M3 series added AV1 decoding support, [ffmpeg does not support it yet](https://trac.ffmpeg.org/ticket/10642).
+
+### Performance Consideration
+
+An Apple Silicon-based Mac is preferred in most cases. Even an entry-level M1 can handle three 4K 24fps Dolby Vision HEVC 10-bit transcoding tasks simultaneously while performing tone-mapping to SDR.<sup>1</sup> <sup>2</sup>
```

Comment:
What are the cases that Apple silicon macs aren't preferred?

Reply:
In a very marginal case, such as someone owning a high-end Intel Mac Pro with multiple GPUs added and disregarding power consumption entirely, it's possible that such a setup might perform better on multi-session transcodes and tone-mapping. However, since I haven't tested this kind of hardware at all, I'm not very sure about its performance. There are also other possible edge cases, such as someone wanting to use the CPU based encoding for specific reasons, where CPU-based HEVC encoding is much better supported on x86_64 platforms. Additionally, there may be other cases that I cannot think of at the moment. Those cases are too edge that I don't even want to document them here.

Reply:
Fair